iosHackthonApp/Modules/Features/SignUp/Sources/SignUp/SignUpWithEmailViewModel.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  SignUpWithEmailViewModel.swift
+//  
+//
+//  Created by Raynelle Francisca on 27/11/2020.
+//
+
+import Foundation
+import Strings
+
+public final class SignUpWithEmailViewModel: ObservableObject {
+	@Published var email: String = ""
+	@Published var moveToNextScreen: Bool = false
+	var isLoading = false
+	var signUpWithEmailTitle: String = Strings.SignUp.signUpWithEmailTitle
```

Comment:
Could typealias the `Strings.SignUp` but not a big deal

Reply:
This gives me `Type alias 'Strings' references itself`, if I do `typealias Strings = Strings.SignUp`
Probably since we are using the variable `Strings`